src/index.js

```diff
@@ -27,6 +28,63 @@ function generateCopyTarget(src, dest, rename) {
   }
 }
 
+function generateCopyTargets(src, dest, rename) {
+  return Array.isArray(dest)
+    ? dest.map(destination => generateCopyTarget(src, destination, rename))
+    : [generateCopyTarget(src, dest, rename)]
+}
+
+async function copyFiles(copyTargets, verbose, copyOptions) {
+  if (Array.isArray(copyTargets) && copyTargets.length) {
+    if (verbose) {
+      console.log(green('copied:'))
+    }
+
+    for (const { src, dest } of copyTargets) {
+      await fs.copy(src, dest, copyOptions)
+
+      if (verbose) {
+        console.log(green(`  ${bold(src)} → ${bold(dest)}`))
+      }
+    }
+  } else if (verbose) {
+    console.log(yellow('no items to copy'))
+  }
+}
+
+function watchFiles(targets, verbose, copyOptions) {
+  return targets.map(({ src, dest, rename }) => {
+    async function onChange(matchedPath) {
+      const copyTargets = generateCopyTargets(matchedPath, dest, rename)
+      await copyFiles(copyTargets, verbose, copyOptions)
+    }
+
+    return chokidar.watch(src, { ignoreInitial: true })
```

Comment:
Unfortunately, we can't pass `src` to chokidar. Chokidar v3 uses [anymatch](https://www.npmjs.com/package/anymatch), which uses [picomatch](https://www.npmjs.com/package/picomatch) inside:

```
chokidar -> anymatch -> picomatch
```

This package uses [globby](https://www.npmjs.com/package/globby) inside for glob support. Globby uses [fast-glob](https://www.npmjs.com/package/fast-glob) inside, which uses [micromatch](https://www.npmjs.com/package/micromatch):

```
globby -> fast-glob -> micromatch -> picomatch
```

So, chokidar v3 misses **micromatch** package which adds support for some glob features on top of picomatch. Globs will have different features for copy and watch. Chokidar used micromatch in v2, but switched in v3.

Reply:
Well that's unfortunate... I'll think about it a bit and do some research into options.

Reply:
I've opened an issue in chokidar https://github.com/paulmillr/chokidar/issues/956, maybe only braces extension is missing and we can live with that.

Reply:
Seems to be compatible. I'll add few tests this week and hopefully release a new version after merging this.